lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -307,5 +307,24 @@ function dosomething_kudos_term_is_selected($reportback_item, $term) {
  * @return bool
  */
 function dosomething_kudos_disable_reactions($nid) {
-  return !variable_get('dosomething_reportback_kudos_enabled', FALSE) || dosomething_helpers_get_variable('node', $nid, 'disable_reactions');
+  return dosomething_helpers_get_variable('node', $nid, 'disable_reactions');
+}
+
+/**
+ * Grab all Kudos data required to render the markup for a given reportback fid
+ * @param  int $fid Reportback file id
+ * @param  int $nid Campaign id for this reportback
+ * @return array Values needed to render Kudos markup
+ */
+function dosomething_kudos_get_kudos_data_for_fid($fid, $nid) {
```

Comment:
oh crap i rewrote all the other function calls without but forgot to actually remove it here